.gitignore

```diff
@@ -9,3 +9,4 @@ temp/
 /test-results/
 /playwright-report/
 /playwright/.cache/
+**/.DS_Store
```

Comment:
why was this removed?

Reply:
I accidentally committed it once, I do not think it needs to be in the repo (its not currently on main or dev/unified-sdk. To avoid this happening again I added to .gitignore.